### PR TITLE
Add book, classroom and rental classes

### PR DESCRIPTION
**Pull request by elassis**
# What I've Accomplished 🏁 
- Add Book, Classroom, and Rentals classes ✔️ 
- Add has many relations between classes ✔️ 
- Add belongs to relations between classes ✔️

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (DeliceLydia) [inline book.rb]: - [x] I suggest that `rentals` be `attribute_accessor` as well to be able to access it and modify it

File: rental.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```
Comment (DeliceLydia) [inline rental.rb]: - [x] `book` and `person` should be `attr_accessor` as well to access them and modify them

File: rental.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+  def initialize(date)
+    @date = date
+    @book = book
+    @person = person
+  end
```
Comment (DeliceLydia) [inline rental.rb]: - [x] This method should be initialized with `date`, `book`, `person`

File: person.rb [hunk 4]
```diff
@@ -35,6 +38,11 @@ def can_use_services?
   def correct_name
     @name = @corrector.correct_name(@name)
   end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.person = self
+  end
```
Comment (DeliceLydia) [inline person.rb]: - [x] To create a new `rental` you should send the person itself should be sent as a parameter to create the new rental achieving the 'has-many' associations. to create a new `rental` here should be created with `date`, `book`, and `person` itself

File: book.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```
Comment (DeliceLydia) [inline book.rb]: - [x] `the book` itself should be sent as a parameter to create the new rental achieving the 'has-many' association. you can create a rental with `date`, `person` and `book` itself

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @elassis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! my major highlights are:
- [ ] PR is professional
- [ ] README is descriptive


## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi @elassis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 

Job well done implementing the requested changes🎉🎉🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**